source/localizable/_index_enter_becp.fr.html.erb

```diff
@@ -1,8 +1,12 @@
 <h3>Entrer dans la beCP&nbsp;?</h3>
-<p>Pour entrer dans la beCP, vous devez nous montrer que vous savez programmer et que vous êtes motivé à vous entraîner. Si vous n'est pas encore rentré en cinquième secondaire, nous vous demandons donc de résoudre tous les exercices
-du 1er chapitre sur la plateforme d'entraînement américaine <%= ext_link_to "USACO", "https://train.usaco.org/usacogate" %>.</p>
-<p>Lorsque vous les aurez tous résolus, envoyez-nous un e-mail à <%= mail_to "becp@be-oi.be" %> avec une archive .zip contenant tous vos codes.</p>
-<p>Si vous êtes en cinquième ou sixième secondaire, contactez nous si vous pensez que vous méritez d'intégrer le groupe.
+<p>Pour entrer dans la beCP, vous devez nous montrer que vous savez programmer et que vous êtes motivé à vous entraîner. Si vous n'est pas encore rentré en sixième secondaire, nous vous demandons donc de résoudre une liste de problèmes
+sur le site de <%= ext_link_to "France-IOI", "http://www.france-ioi.org" %> (progression plus lente, site en français) ou de <%= ext_link_to "USACO Guide", "https://usaco.guide" %> (progression plus rapide, site en anglais).</p>
+<p>Pratiquement: Créez un compte sur le site sur le site que vous désirez utiliser et rejoigner le groupe qui se trouve sur <%= ext_link_to "ce lien", "http://www.france-ioi.org/algo/group.php?idGroup=20052" %> pour France-IOI
+ou sur <%= ext_link_to "ce lien", "https://usaco.guide/groups/join?key=rjW0XfoSo8iPeB4GVDNA" %> pour USACO Guide afin que nous puissions suivre votre progression.
+En fontion de l'année dans laquelle vous vous trouvez, une différente quantité d'exercices est attendue de vous.
```

Comment:
fontion -> fonction

Reply:
Fixed. Apparently you need to reapprove again since there was a change.